libse/Utilities.cs

```diff
@@ -754,6 +754,25 @@ public static List<string> GetDictionaryLanguagesCultureNeutral()
             return list;
         }
 
+        public static IEnumerable<CultureInfo> GetSubtitleLanguageCultures()
+        {
+            var excludes = new HashSet<string>();
+            foreach (var ci in CultureInfo.GetCultures(CultureTypes.NeutralCultures))
+            {
+                if (ci.Name.Length < 4 && ci.Name == ci.IetfLanguageTag)
+                {

```

Comment:
You could add the "ci" to a "temp list here" so we don't have to call "CultureInfo.GetCultures(CultureTypes.NeutralCultures)" twice.

Reply:
Done.